libs/service-portal/vehicles/src/screens/VehicleDetail/VehicleDetail.tsx

```diff
@@ -129,7 +129,7 @@ const VehicleDetail: ServicePortalModuleComponent = () => {
         />
         <Divider />
 
-        {/* <UserInfoLine
+        <UserInfoLine
```

Comment:
In `libs/service-portal/vehicles/src/components/DetailTable/FeeInfoItem.tsx` as well 😃

Reply:
AH great catch 💥